ext/ldap/ldap.stub.php

```diff
@@ -260,9 +260,9 @@ function ldap_parse_result($link, $result, &$errcode, &$matcheddn = null, &$errm
 #if defined(LDAP_API_FEATURE_X_OPENLDAP) && defined(HAVE_3ARG_SETREBINDPROC)
 /**
  * @param resource $link
- * @param callable $callback
+ * @param callable|null $callback
```

Comment:
I've just noticed that this PHPDoc should be there anymore

Reply:
Do you mean this single line, this doc block or all the doc block in ldap.stub.php?

Reply:
Sorry, I meant just this line, since the type info is superfluous now

Reply:
Force-pushed new version with the line removed.

Reply:
Thanks! I'll merge the PR soon